app/src/main/java/com/example/maciej/smim/Game.kt

```diff
@@ -99,5 +99,62 @@ open class Game(numberOfRows: Int, numberOfColumns: Int) {
         return false
     }
 
+    fun checkFields(numberOfButton: Int, playerNumber: Int, toWin: Int = 5): Boolean{
```

Comment:
Is it functionally the same as the original one? If so, absolutely DO NOT rewrite it. Add an optional argument and use `if` statements where you need them.
https://kotlinlang.org/docs/reference/functions.html#default-arguments